app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def create
+    puts params
+    @comment = current_user.comments.new(params.require(:comment).permit(:text))
```

Comment:
- [ OPTIONAL ] I think it would be great if you make the parameters private in a method to make them a strong parameter you can check this [lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) to learn more about it ✨🔥